build.gradle

```diff
@@ -81,6 +82,34 @@ task systemtest(dependsOn: 'zipReport', type: Test) {
   }
 }
 
+task startServerInBackground(dependsOn: 'classes', type: com.github.psxpaul.task.JavaExecFork) {
+  main = mainClassName
+  classpath = sourceSets.main.runtimeClasspath
+  args = ['--repo', 'https://github.com/reposense/RepoSense.git', '--view']
+  waitForPort = 9000
+}
+
+task installCypress(type: com.liferay.gradle.plugins.node.tasks.NpmInstallTask) {
+  workingDir = file('frontend/cypress/')
+}
+
+task cypress(dependsOn: 'installCypress', type: com.liferay.gradle.plugins.node.tasks.ExecuteNpmTask) {
```

Comment:
@yong24s Sorry missed this, can make `cypress` task depend on `startServerInBackground` as well, so we don't need to separate running a server for this tes